lib/SILOptimizer/IPO/DeadFunctionElimination.cpp

```diff
@@ -280,11 +279,22 @@ class DeadFunctionAndGlobalElimination {
   }
 
   /// Marks a witness table as alive if it is not alive yet.
-  void ensureAliveConformance(const ProtocolConformance *C) {
-    SILWitnessTable *WT = Module->lookUpWitnessTable(C);
-    if (!WT || isAlive(WT))
-      return;
-    makeAlive(WT);
+  void ensureAliveConformance(ProtocolConformanceRef conformance) {
+    if (conformance.isConcrete()) {
+      auto *concrete = conformance.getConcrete();
+      SILWitnessTable *WT = Module->lookUpWitnessTable(concrete);
+      if (!WT || isAlive(WT))
+        return;
+      makeAlive(WT);
+
+      // FIXME: If 'concrete' is a SpecializedProtocolConformance,
```

Comment:
@eeckstein Thoughts?

Reply:
It actually doesn't matter because we don't do dead witnesstable elimination. In `findAnchorsInTables` we set all witness tables alive. We had to do this because it's almost impossible to find all references to witness tables in SIL. Only IRGen knows that - and we emit witness tables lazily there.

Reply:
Ah. I'll just delete the logic then since it's pointless to keep around code that is unused and incorrect.